apps/aecore/lib/aecore/account/tx/spend_tx.ex

```diff
@@ -74,6 +76,9 @@ defmodule Aecore.Account.Tx.SpendTx do
       length(senders) != 1 ->
         {:error, "#{__MODULE__}: Invalid senders number"}
 
+      !is_binary(tx.payload) ->
+        {:error, "#{__MODULE__}: Invalid payload "}
```

Comment:
include invalid data in error message, this checks if it is not binary, make the message descriptive